addons/pos_epson_printer/models/pos_config.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    epson_printer_ip = fields.Char(string='Epson Printer IP', help="Local IP address of an Epson receipt printer.")
+    epson_printer_cashdrawer = fields.Boolean(help="Automatically open the cashdrawer.")
```

Comment:
do we need this field, can't we just use 'iface_cashdrawer' maybe see with lap for the usability

Reply:
I've just checked, you can put the same field twice in a view. When you check one the other will be checked too. I'll remove `epson_printer_cashdrawer` and put the same field in both places.